app.py

```
@@ -7,6 +7,9 @@
 from sklearn.preprocessing import LabelEncoder
 import os
 from pathlib import Path
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)